sentinel-core/src/main/java/com/alibaba/csp/sentinel/log/LogBase.java

```diff
@@ -33,9 +36,20 @@
     private static final String DIR_NAME = "logs" + File.separator + "csp";
     private static final String USER_HOME = "user.home";
     public static final String LOG_DIR = "csp.sentinel.log.dir";
+    public static final String LOG_NAME_USE_PID = "csp.sentinel.log.use.pid";
+    private static boolean logNameUsePid = false;
+
     private static String logBaseDir;
 
     static {
+        try {
+            init();
+        } catch (Throwable t) {
+            t.printStackTrace();
```

Comment:
Once an error is thrown (e.g. class initialization error due to no permission), the application might need to fail immediately, as it may cause weird behaviors (e.g. NPE when publishRecord).